paddle/fluid/framework/ir/graph.cc

```diff
@@ -17,10 +17,110 @@ limitations under the License. */
 #include "paddle/fluid/framework/ir/graph.h"
 #include "paddle/fluid/framework/operator.h"
 
+DEFINE_bool(ssa_program, false, "Convert all blocks in program into SSAgraphs");
```

Comment:
`ssa_graph` may not be a good name. You can name as what it does: `convert_subblocks`

Reply:
I'll rename 'ssa_program'.